### PR TITLE
feat: close button on modal and drawer

### PR DESCRIPTION
**Pull request by jeromeraffin**
https://user-images.githubusercontent.com/19819523/140331550-3b2f6d2b-ab07-4d90-bd50-e0a8319c6ed9.mp4

### CODE REVIEW

**Comment by jeromeraffin** [comment]
https://user-images.githubusercontent.com/19819523/140331318-9b5d2651-8e5f-40d0-8399-681470579830.mp4

File: packages/CloseButton/styles.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import styled, { css, keyframes } from '@xstyled/styled-components'
+import { th } from '@xstyled/system'
+import { Button } from '@welcome-ui/button'
+import { system } from '@welcome-ui/system'
+import { hexToRGBA } from '@welcome-ui/utils'
+
+const fade = keyframes`
```
Comment (theo-mesnil) [inline packages/CloseButton/styles.js:7]: I think is useless, it was only on ClearButton to add a good effect on button apparition.

File: packages/CloseButton/styles.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  to {
+    opacity: 1;
+  }
+`
+
+const fadeRule = css`
+  ${fade}
+`
+
+export const CloseButton = styled(Button)(
```
Comment (theo-mesnil) [inline packages/CloseButton/styles.js:20]: Maybe we can create a theme entry, to customize this button for another people

File: packages/Modal/styles.js [hunk 3]
```diff
@@ -68,7 +68,7 @@ export const Dialog = styled(ReakitDialog)(
 )
 
 export const Content = styled(Box)`
-  padding: xxl;
+  padding: 50;
```
Comment (theo-mesnil) [inline packages/Modal/styles.js:71]: todo for the v4, add 50 to new space '5xl'

Comment (Calvein) [reply packages/Modal/styles.js:71]: Yes please 😇

**Review by theo-mesnil — APPROVED** [review]

**Review by mleralec — APPROVED** [review]

**Pull request merged**